.github/workflows/preview-env-deploy.yml

```diff
@@ -55,6 +74,15 @@ jobs:
           env: ${{ github.event.repository.name }}
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: Update comment with upload status
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          comment-id: ${{ steps.find-comment.outputs.comment-id }}
+          issue-number: ${{ github.event.number }}
+          body: |
+            The preview environment relating to the commit ${{ env.short_sha }} is being uploaded to the bucket. <!-- preview-env -->
```

Comment:
Small suggestion for providing more context to the users

```suggestion
            :arrow_up: The preview environment build results for commit ${{ env.short_sha }} is being uploaded. This usually takes 2-3 minutes. <!-- preview-env -->
```

Reply:
(I've removed the `bucket` mention because I don't want to expose implementation details here, it's an open source project; contributors can figure this out when looking in the code, but we should not be openly mentioning it in the comment - same as hiding the bucket name in the secret)

Reply:
Yeah this is actually a really good point